JetAnalyzers/python/Defaults_cff.py

```diff
@@ -25,14 +25,14 @@
 JetResponseParameters = cms.PSet(
     # record flavor information, consider both RefPt and JetPt
     doComposition   = cms.bool(True),
-    doFlavor        = cms.bool(True),
+    doFlavor        = cms.bool(False),
     doRefPt         = cms.bool(True),
     doJetPt         = cms.bool(True),
     saveCandidates  = cms.bool(False),
     # MATCHING MODE: deltaR(ref,jet)
-    deltaRMax       = cms.double(0.25),
+    deltaRMax       = cms.double(0.4),
```

Comment:
I've been out of the loop for a while, so feel free to tell me I don't know the latest developments. That said, `0.4` seems awfully permissive for the deltaR matching. That's the entire cone radius for an `R=0.4` jet. The default here should be the most often used value and should not change much. Is that why you changed this to `0.4`.

Reply:
Apologies for the confusion, this was actually a test version and we will fix it and upload the correct one with the values that were before. (However, this does not affect the results because we are using dr max value of 0.2 in the MC Truth JECs derivation.)